### DISCUSSION TITLE
When Astroid 3.0.6 will be compatible with JOOMLA 5.0.1 or more

### DISCUSSION THREAD

**Discussion by jungdan57** [templaza/astroid-framework#485] [Q&A] [answered]
Hello,

I have update my JOOMLA WEB site to JOOMLA 5.0.1
I try to install the ultimate Astroid Framework : astroid-framework-3.0.6.zip with the installation process from JOOMLA.
Extensions ==> Gestion ==> Installation ==> Install from a file 
The file : astroid-framework-3.0.6.zip was in the good directory \joomla/tmp and the xml file was in the sub directory astroid-framework-3.0.6.

When the install program finished, i had the message : bibliotheque : installation effectuée (my JOOMLA is in French).

But later, i can't see the front 'site' template.

Not good ...

When will it be possible to download an Astroid Framework version full compatible with JOOMLA 5.xx ?

Thanks for your answer.

And i hope it will be soon for the compatible PACK Astroid for JOOMLA 5, because the history have change to JOOMLA 5 yet.

**Comment by sonvnn** [comment, marked as answer]
Astroid 3 is full compatible with Joomla 5 even when you disable Backward Compatible plugin. To install full package includes Core and Template, please download Template package https://github.com/templaza/astroid-framework/releases/download/v3.0.6/astroid-package-v3.0.6.Framework-Template.zip

After that go to System >> Template Styles >> Choose your template (One or Zero) >> Set default template >> Done

Please let me know if you have further questions.

Best Regards,
Sonny

**Reply by jungdan57** [reply]
Hello,

I have disabled Backward Compatible plugin, and it is better, but i have found various critical BUGS.


The First One : If you make a copy from de Template one
It is not the same that before with JOOMLA 3 or JOOMLA 4

Template : Astroid_template_one Détails et fichiers

Clic : Astroid_template_one Détails et fichiers

Create a child template with the name : OAD 
And the Styles of templates : astroid_template_one – Par défaut

Now we have a new Template named :
Astroid_template_one_oad Détails et fichiers 

In JOOMLA 3 or JOOMLA 4 the Child Template was named with this creation of child template :
oad Détails et fichiers


This action of the creation of a child Template will create Styles Templates :

In JOOMLA 5 :
The new Child Template named : 
Astroid_template_one_oad Détails et fichiers 
Create 2 Styles Templates :
astroid_template_one_oad - Par défaut
and another one with the name :
Astroid_template_one_oad, copie de astroid_template_one - Par défaut

Why doesn’t create just one Style named oad

In JOOMLA 3 or JOOMLA 4 :
The child Template : oad Détails et fichiers
Create just one style named : oad



And now the second problem :
In JOOMLA 5 i look to use what it was created.

The new Child Template named : 
Astroid_template_one_oad Détails et fichiers 
2 Styles Templates :
astroid_template_one_oad - Par défaut
and another one with the name :
Astroid_template_one_oad, copie de astroid_template_one - Par défaut

I choose to rename the style :
astroid_template_one_oad - Par défaut
to
oad - Par défaut

No problem for rename.
And i select the new named oad – Par défaut to use it 

Next, i use SP TRANSFER to transfer the tables JOOMLA 3.10.12 to JOOMLA 5.0.1 (PHP 8.2.11, MYSQL 8.2.0)
For do that, i reactive Backward Compatible plugin, the import is OK.

After the import i disable Backward Compatible plugin for using Astroid 3.0.6 in JOOMLA 5.0.1

 
Now, i have the content of JOOMLA 3.10.12 into JOOMLA 5.0.1 and Astroid 3.0.6 installed with the Style renamed before like :
oad - Par défaut

i go to Styles Template, and clic :
oad - Par défaut

Next go to : menu assignements
Nothing selected
I selected : inverse all selections or inverse the selection
And valide OK.

The selections don’t stay like before.

It is imposible to assign all the lines menu to the style Template.


When i select just one line menu, then validate, the selection stay good.
But next if i select another one, just the first selection stay selected when i validate.


There is a big problem to assign the lines menus to a style template.

Could you take a look to resolve this problem.

Sorry for my approximative english, i hope you can understand me in my explications.

**Reply by sonvnn** [reply]
Thank you for feedback! May you make a video record about your issues? So it is easy for me to understand what you did.

Thanks & Best Regards,
Sonny

**Reply by jungdan57** [reply]
I can't make a video sorry

Just try to create a Child Template from Template One Astroid 3.0.6 in JOOMLA 5.0.1 (PHP 8.2.11, MYSQL 8.2.0)
and i think you will have the same : 2 creations of Styles Templates.

( I create a child Template to not have to use the original ONE, and it was easier for me to have the same parameters i had in JOOMLA 3. The best would be to have the same Template Name and Style Template name like in the JOOMLA 3 version.)

And now rename one of these, and look if it will posible to assign existing menus.

For me that's imposible.

Thanks for help
